allennlp/common/tee_logger.py

```diff
@@ -10,7 +10,7 @@ class TeeLogger:
         sys.stdout = TeeLogger("stdout.log", sys.stdout)
```

Comment:
@joelgrus This is nasty - we are printing to this instead of `sys.stdout` so that we can save a copy. `sys.stdout` has type `io.TextIOWrapper`, but when I try to subclass this, it takes a `buffer` parameter in its constructor which doesn't appear in the actual `io` library. I'm a little out of my depth ....